src/components/seeds/database/index.tsx

```diff
@@ -423,86 +424,80 @@ export default function Database(props: DatabaseProps): JSX.Element {
     void populatePendingAccessions();
   }, [selectedOrganization.id]);
 
-  const initAccessions = useCallback(
-    (activeRequests: boolean) => {
-      const getFieldsFromSearchColumns = () => {
-        let columnsNamesToSearch: string[];
-        if (searchColumns.includes('active')) {
-          columnsNamesToSearch = [...searchColumns, 'id'];
-        } else {
-          columnsNamesToSearch = [...searchColumns, 'active', 'id'];
-        }
-
-        return columnsNamesToSearch;
-      };
+  const initAccessions = useCallback(() => {
```

Comment:
This looks like a lot of changes but it's only indentation (removed the useCallback argument).
Look for get/setRequestId, those are the changes.